### PR TITLE
add "contact" section to docs

### PR DESCRIPTION
**Pull request by iusiri**

### CODE REVIEW

File: docs/docs/contactus.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: docs
+title: Contact Us
+---
+
+
+# Contact Us
+Want to learn more from the Refocus team? Contact us at: cxtools at salesforce.com
```
Comment (iamigo) [inline docs/docs/contactus.md]: I'm sure you're not using an actual email address for a reason... but why?

Also, we don't really want folks emailing us for bugs and feature requests--we want them to create Issues... that's why I had that stuff in the example I sent you.

**Comment by iamigo** [comment]
@iusiri

**Comment by iusiri** [comment]
@iamigo - updated

**Comment by iusiri** [comment]
Also -- the weird email is to avoid web crawlers.

File: docs/docs/contactus.md [hunk 2]
```diff
@@ -5,4 +5,12 @@ title: Contact Us
 
 
 # Contact Us
-Want to learn more from the Refocus team? Contact us at: cxtools at salesforce.com
\ No newline at end of file
+
+##Found a bug or have a great feature idea?
+Use Issues https://github.com/salesforce/refocus/issues.
```
Comment (iamigo) [inline docs/docs/contactus.md]: `Use [Issues](https://github.com/salesforce/refocus/issues).`

File: docs/docs/contactus.md [hunk 3]
```diff
@@ -5,4 +5,12 @@ title: Contact Us
...
 
 # Contact Us
-Want to learn more from the Refocus team? Contact us at: cxtools at salesforce.com
\ No newline at end of file
+
+##Found a bug or have a great feature idea?
+Use Issues https://github.com/salesforce/refocus/issues. 
+
+##Want to contribute to Refocus?
+Here're our contribution guidelines: https://salesforce.github.io/refocus/docs/95-contributing.html
```
Comment (iamigo) [inline docs/docs/contactus.md]: `Check out our [contribution guidelines](https://salesforce.github.io/refocus/docs/95-contributing.html).`

File: docs/docs/contactus.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+layout: docs
+title: Contact Us
+---
+
+
+# Contact Us
+
+##Found a bug or have a great feature idea?
+Use Issues https://github.com/salesforce/refocus/issues.
```
Comment (iamigo) [inline docs/docs/contactus.md]: square brackets around the word "Issues" then no space then parentheses around the url

**Comment by iusiri** [comment]
@iamigo - please review.

**Pull request merged**